### PR TITLE
Improve the documentation of the README file

### PR DESCRIPTION
**Pull request by clementprdhomme**
**⚠️ This PR assumes #89 has been merged already.**

This PR improves the documentation of the README file:

- Technological stack
- Updated instructions to start developing
- Documentation of the environment variables
- Description of the CI/CD
- Vulnerability mitigation

## Testing instructions

Have a read to make sure the information is correct.

## Tracking

Not tracked.

### CODE REVIEW

File: frontend/README.md [hunk 1]
```diff
@@ -1,117 +1,75 @@
...
+## Environment variables
 
-Before running the application, you need to configure it by copying `.env.example` to `.env` and setting the appropriate values where needed.
+The application is configured via environment variables stored in a `.env` file that must be placed at the root of the project. You can create one by copying `.env.example` and setting a value for each key.
 
-To start the application, run:
+Below is a description of each of the keys.
 
-    yarn dev
+| Variable               | Description                                                                                                                                       |
```
Comment (clementprdhomme) [inline frontend/README.md:36]: @davidsingal I think we could reduce the number and rename the environment variables. Is this something you'll do in #89?

File: frontend/README.md [hunk 2]
```diff
@@ -1,117 +1,75 @@
...
+|------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------|
+| `NEXT_PUBLIC_SITE`     | Complete URL from which the front-end application is served (including https) and without a trailing slash (e.g. https://www.resilienceatlas.org) |
+| `NEXT_PUBLIC_API_HOST` | Complete URL of the API server (including https) and without the trailing slask (e.g. https://www.resilienceatlas.org)                            |
+| `NEXT_PUBLIC_API_PROD` | Same as NEXT_PUBLIC_API_HOST                                                                                                                      |
 
-## Production build
+## CI/CD
```
Comment (clementprdhomme) [inline frontend/README.md:42]: @davidsingal Can you make sure this section is accurate?

File: frontend/README.md (hunk 1 above)
Comment (davidsingal) [reply frontend/README.md:36]: Yes, only `NEXT_PUBLIC_API_HOST` will be required

File: frontend/README.md (hunk 2 above)
Comment (davidsingal) [reply frontend/README.md:42]: It is

**Review by davidsingal — APPROVED** [review]

File: frontend/README.md (hunk 1 above)
Comment (clementprdhomme) [reply frontend/README.md:36]: I removed the other two environment variables.

Make sure to replace `REACT_APP_SITE` in your branch and to update https://github.com/ConservationInternational/resilienceatlas/blob/develop/.github/workflows/frontend_test.yml#L63-L65.

Comment (davidsingal) [reply frontend/README.md]: Updated!

**Pull request merged**